lib/rubocop/cop/metrics/line_length.rb

```diff
@@ -40,6 +45,30 @@ def max
           cop_config['Max']
         end
 
+        def allow_heredoc?
+          allowed_heredoc
+        end
+
+        def allowed_heredoc
```

Comment:
When you set the option to a list of allowed heredoc delimiters, you need to check whether the heredoc containing the long line is whitelisted. Thus this method is required. One can argue about `#allow_heredoc?` above, which actually *just* returns whether the config is not false. I added `#allow_heredoc?` so that in `#investigate` it looks balanced with `#allow_uri?`. Let me know what the preference is here.